Multisig/Assets.xcassets/ico-remove.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
```

Comment:
I think there should be a system image for this icon. Did you take a look?

Reply:
The system icon is noticeably thinner that's why I used the designed one

Reply:
Have you tried the `Image(systemName: "...").font(Font.body.bold())` to make it thick?